datatables.go

```diff
@@ -497,6 +497,14 @@ func (p *Parser) bindGameRules() {
 		entity.BindProperty(grPrefix("m_totalRoundsPlayed"), &p.gameState.totalRoundsPlayed, st.ValTypeInt)
 		entity.BindProperty(grPrefix("m_bWarmupPeriod"), &p.gameState.isWarmupPeriod, st.ValTypeBoolInt)
 
+		entity.FindProperty(grPrefix("m_bHasMatchStarted")).OnUpdate(func(val st.PropertyValue) {
+			p.eventDispatcher.Dispatch(events.MatchStartChanged{
+				OldIsStarted: p.gameState.isMatchStarted,
+				NewIsStarted: val.IntVal == 1,
+			})
+			p.gameState.isMatchStarted = val.IntVal == 1
```

Comment:
Move this to before the dispatching, to avoid `GameState.MatchStarted()` returning the wrong value inside an event-handler.